tests/integration/cli/segmentation/test_segmentation.py

```diff
@@ -219,30 +214,8 @@ def test_otx_multi_gpu_train(self, template, tmp_dir_path):
         otx_train_testing(template, tmp_dir_path, otx_dir, args1)
 
 
-# tmp: create & remove data.yaml to only use train-data-roots
-def set_dummy_data(func):
-    @wraps(func)
-    def wrapper(*args, **kwargs):
-        # create data.yaml
-        to_save_data_args = {
-            "data": {
-                "train": {"ann-files": None, "data-roots": None},
-                "val": {"ann-files": None, "data-roots": None},
-                "unlabeled": {"file-list": None, "data-roots": None},
-            },
-        }
-        yaml.dump(to_save_data_args, open("./data.yaml", "w"), default_flow_style=False)
-        # run test
-        func(*args, **kwargs)
-        # remove data.yaml
-        os.remove("./data.yaml")
-
-    return wrapper
-
-
 args_selfsl = {
-    "--train-ann-file": "data/segmentation/custom/annotations/detcon_masks",
-    "--train-data-roots": "data/segmentation/custom/images/training",
+    "--train-data-roots": "data/datumaro/common_semantic_segmentation_dataset/train",
```

Comment:
@sungchul2 , Could you double-check this changed part? 

After Datumaro integration, temporary function named `set_dummy_data` is not needed anymore. So, I removed it, and changed `--train-data-roots`.

Reply:
+same here: https://github.com/openvinotoolkit/training_extensions/pull/1390/files#diff-ae17c3fa6c705014cd42d211f9a70b36213e6f9478da54ec9f5b498782ee0708R547

Reply:
In my test cases, there is no problem :)

Reply:
I temporarily set this decorator to only use `--train-data-roots` (and `--train-ann-file` for seg) on CLI.
If everything is okay, it can be removed :D
Thanks for sharing!